Shared/Supporting Files/Views/CategoryGridView.swift

```diff
@@ -38,8 +38,11 @@ struct CategoryGridView: View {
             LazyVGrid(columns: columns) {
                 ForEach(categories, id: \.self) { category in
                     NavigationLink {
-                        SampleListView(samples: samples.filter { $0.category == category })
-                            .navigationTitle(category)
+                        List(samples.filter { $0.category == category }, id: \.name) { sample in
+                            SampleRow(sample: sample)
+                        }
+                        .listStyle(.sidebar)
+                        .navigationTitle(category)
```

Comment:
Methinks this should be a view named `CategoryView`:

```swift
struct Category {
    let name: String
    let samples: [Sample]
}

struct CategoryView: View {
    let category: Category

    var body: some View {
        List(category.samples, id: \.name) { sample in
            SampleRow(sample: sample)
        }
        .listStyle(.sidebar)
        .navigationTitle(category.name)
    }
}
```

Reply:
I named it `CategoryList` for now.